YouTubeVideoSearch/src/index.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import { Provider } from "react-redux";
+import { createStore, applyMiddleware } from "redux";
+
+import thunk from "redux-thunk";
+import { composeWithDevTools } from "redux-devtools-extension";
+
+import searchReducer from "./reducers/search";
+
+import App from "./components/App";
+
+// import { fetchYoutubeVideo } from "./service/service";
+
+import "./styles/App.scss";
+import "./styles/search-form.scss";
+import "./styles/video-info.scss";
+import "./styles/videos-container.scss";
```

Comment:
Usually we import style file inside the component where is it used.

There are two common practices for React project file structure.
1)  One common way to structure projects is locate CSS, JS, and tests together, grouped by feature or route.
2)  Another popular way to structure projects is to group similar files together.